kafka-ui-react-app/.eslintrc.json

```diff
@@ -33,7 +33,10 @@
     "jsx-a11y/label-has-associated-control": "off",
     "import/prefer-default-export": "off",
     "@typescript-eslint/no-explicit-any": "error",
-    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }]
+    "import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
+    "import/no-cycle": "error",
+    "import/order": ["error", {"groups": ["builtin", "external", "parent", "sibling", "index"]}],
```

Comment:
🥳 
I also like `{"newlines-between": "always"}` option. But I'm not strong on this if you think this is too much.